hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/web/ozShell/token/PrintTokenHandler.java

```diff
@@ -18,54 +18,31 @@
 
 package org.apache.hadoop.ozone.web.ozShell.token;
 
-import org.apache.hadoop.ozone.OzoneSecurityUtil;
-import org.apache.hadoop.ozone.web.ozShell.Handler;
+import org.apache.hadoop.ozone.security.OzoneTokenIdentifier;
 import org.apache.hadoop.ozone.web.utils.JsonUtils;
 import org.apache.hadoop.security.token.Token;
 import picocli.CommandLine;
 import picocli.CommandLine.Command;
 
-import java.nio.charset.StandardCharsets;
-import java.nio.file.Files;
-import java.nio.file.Paths;
+import java.util.concurrent.Callable;
 
 /**
- * Executes getDelegationToken api.
+ * Prints token decoded from file.
  */
 @Command(name = "print",
     description = "print a delegation token.")
-public class PrintTokenHandler extends Handler {
+@SuppressWarnings("squid:S106") // CLI
+public class PrintTokenHandler implements Callable<Void> {
```

Comment:
Am I understand well that this one is on purpose not a Handler implementation, as it is dealing with a local client side thing, and there is no need for the facilities that are there to connect to Ozone server side?

Reply:
Correct.

Reply:
great, thank you for clarifying